packages/platforms/delta/arm64/tn48m/src/arm64_delta_tn48m/module/src/platform_lib.h

```diff
@@ -40,6 +40,7 @@ typedef enum plat_id {
     PID_TN4810M_PVT,
     PID_TN48M2,
     PID_TN4810M_NONPVT,
+    PID_TN48M2_SWDEV,
```

Comment:
Sort it (below `PID_TN48M2`)?

Reply:
Most of the IDs are fixed by HW, its not suitable to disarray them.